lib/opsworks.coffee

```diff
@@ -0,0 +1,53 @@
+# Wrapper around the AWS OpsWorks sdk, built around our specific
+# requirements
+
+AWS = require('aws-sdk')
+Q = require('q')
+
+access = process.env.AWS_ACCESS_KEY_ID
+secret = process.env.AWS_SECRET_ACCESS_KEY
+region = process.env.AWS_REGION
+
+AWS.config.update(accessKeyId: access, secretAccessKey: secret)
```

Comment:
Ah, good point; I regularly forget about this. We should be able to use IAM roles instead ([per the SDK docs](http://docs.aws.amazon.com/AWSJavaScriptSDK/guide/node-configuring.html#Credentials_from_IAM_Roles_for_EC2_Instances)). Hard to develop with IAM roles locally, though.